components/builder-api/src/config.rs

```diff
@@ -109,6 +109,8 @@ pub struct ApiCfg {
     pub build_on_upload:  bool,
     pub private_max_age:  usize,
     pub saas_bldr_url:    String,
+    #[serde(with = "deserialize_into_vec")]
```

Comment:
Will add and use a similar function like deserialize_into_vec.  
IMO deserialize_into_vec should have something in its name to indicate that it is changing the case.
Like:
deserialize_into_vec_upper

Reply:
I don't think you need a deserializer. I think a `Vec<String>` should work as is. Like `bootstrap_nodes` in https://github.com/habitat-sh/builder/blob/5298517e6da96ccc8b2c6029c9dc63bf937e312d/components/builder-events/src/kafka.rs#L40

Reply:
OK

Reply:
right it would need to be a proper toml array. That format is in line with other config we have. If you wanted to stick with "origin1, origin2" I think you would need to create a deserializer.

Reply:
OK - I think we should stick to whatever convention we are already using.